core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -110,7 +115,9 @@ public FileInStream openFile(URIStatus status, OpenFilePOptions options)
               .build();
       return mDoraClient.getInStream(status, openUfsBlockOptions);
     } catch (RuntimeException ex) {
-      LOG.debug("Dora client open file error. Fall back to UFS.", ex);
+      UFS_FALLBACK_COUNTER.inc();
```

Comment:
How about the GetStatus()?

Reply:
Hi @huanghua78, thanks for your suggestion!

Are you referring to the UFS fallback in the `getStatus()` function? I updated the PR and added the capture of the UFS fallback there with the same metric (We can have another metric for the `getStatus()` if necessary). Feel free to have a look and let me know your thought.

Reply:
Yes, that's what I meant.
Looks good.